.github/workflows/e2e.yml

```diff
@@ -1,11 +1,6 @@
 name: E2E
 
 on:
-  push:
```

Comment:
Since the `push` trigger is being removed, should we also add a `cron` trigger to periodically run the E2E tests on main?

Reply:
Not now, the E2E tests are currently broken and need to be refactored to do not depend on AIS or IoT Hub at all and instead test at MPI level.

Reply:
Eventually yes, the e2e are not working because of the broken AIS dependency